ios/RNRadar.m

```diff
@@ -787,10 +787,16 @@ - (void)didLogMessage:(NSString *)message {
         country = optionsDict[@"country"];
     }
 
+    BOOL expandUnits = false;
+    NSNumber *expandUnitsPointer = optionsDict[@"expandUnits"];
+    if (expandUnitsPointer != nil && [expandUnitsPointer isKindOfClass:[NSNumber class]]) {
+        expandUnits = [expandUnitsPointer boolValue]; 
+    }
```

Comment:
look at `RadarUser.m` or a few lines above for more idiomatic naming. Probably want to change `expandUnitsPointer` to `expandUnitsNumber` (or `expandUnitsObj`).